model/src/main/kotlin/Mappers.kt

```diff
@@ -72,4 +72,14 @@ inline fun <reified T> String.fromYaml(): T = yamlMapper.readValue(this)
 
 fun Any?.toYaml(): String = yamlMapper.writeValueAsString(this)
 
+fun Any?.toJson(prettyPrint: Boolean = true): String {
```

Comment:
You're arguing to be analog to `toYaml`, but the latter implicitly defaults to `prettyPrint = false` (as it does not take that argument at all), but this new functions defaults to `prettyPrint = true` for JSON. So how about changing the default to `prettyPrint = false` here as well for consistency?

On the other hand, I wonder what pretty-printing YAML actually means, as you can't put everything into a single line with YAML anyway...

Reply:
Alternative could be `toPrettyJson()`.

Reply:
> What do you prefer?

Code-wise I believe it can stay as-is, but I'd appreciate an addition to the commit message saying that `prettyPrint = true` was chosen as the default because 1. that better matches the YAML output which is "inherently pretty", 2. as you say, that function is primarily used in tests where pretty JSON is easier to read in diffs.